department-of-veterans-affairs/va.gov-cms
# VACMS-4191: Count nodes per section for VAMC scalability research.

## Description

See #4191. 

This view produces a report, mainly for internal purposes, although it may be helpful to VAMC upgrade and Facilities teams. 

## Testing done

Manual/visual. 

## Screenshots

![Nodes_per_section___VA_gov_CMS](https://user-images.githubusercontent.com/643678/106904298-07305600-66c9-11eb-9395-672502529fc3.png)


## QA steps

As content admin or admin, go to /admin/content/audit/count-nodes-by-section

Poke around a bit, make sure it makes sense. If you click on a parent right now (eg VISN 15), you have to remove the last `/all` argument to see the children (/admin/content/audit/count-nodes-by-section/VISN%2015)

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
